src/backend/utils/__mocks__/indexer.js

```diff
@@ -20,7 +20,13 @@ const deletePost = (postId) => {
   return Promise.resolve();
 };
 
-const search = () => Promise.resolve(db);
+const search = (keyword = '') => {
+  const filtered = db.values.filter((e) => e.text.includes(keyword));
+  return Promise.resolve({
+    results: filtered.length,
+    values: filtered.map((e) => ({ id: e.id })),
```

Comment:
I think you can change this to `values: filtered.map((value) => ({ id })),`

Reply:
If I do it, I got the error.
![image](https://user-images.githubusercontent.com/59578350/98194426-65dd0c80-1eed-11eb-834e-6ead898455af.png)